### ISSUE TITLE
Instaling on CentOS

### ISSUE THREAD

**Issue by mtopper** [pglombardo/PasswordPusher#49]
I'm receiving the following when trying to run bundle install --without development production test --deployment

The git source `git://github.com/pglombardo/ezcrypto.git` uses the `git` protocol, which transmits data without encryption. Disable this warning with `bundle config git.allow_insecure true`, or switch to the `https` protocol to keep your data secure.
The git source `git://github.com/russfrisch/modernizr-rails.git` uses the `git` protocol, which transmits data without encryption. Disable this warning with `bundle config git.allow_insecure true`, or switch to the `https` protocol to keep your data secure.
You are trying to install in deployment mode after changing
your Gemfile. Run `bundle install` elsewhere and add the
updated Gemfile.lock to version control.

You have added to the Gemfile:
- pry (= 0.9.12.4)

You have deleted from the Gemfile:
- pry
- pry-byebug

**Comment by pglombardo** [comment]
Hmm that's interesting.  The quick fix is to change the Gemfile to use https.  Replace the ezcrypto and modernizr-rails gem lines with the following:

```
gem 'ezcrypto', :git => 'https://github.com/pglombardo/ezcrypto.git'
gem 'modernizr-rails', :git => 'https://github.com/russfrisch/modernizr-rails.git'
```

Then run `bundle install --without development production` and then again with the `--deployment` flag.

For the longer term, maybe I should change that to HTTPS by default.

**Comment by mtopper** [comment]
Thanks!

**Comment by CodeGnome** [comment]
This issue would be closed by merging #53.

**Comment by pglombardo** [comment]
Agree - @mtopper if there is any further issues, feel free to re-open this or add a new issue.
